pkg/webhook/config.go

```diff
@@ -318,16 +320,38 @@ func parseVaultConfig(obj metav1.Object, ar *model.AdmissionReview) VaultConfig
 		vaultConfig.AgentOnce = false
 	}
 
-	if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-cpu"]); err == nil {
-		vaultConfig.AgentCPU = val
+	if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-cpu-limit"]); err == nil {
+		vaultConfig.AgentCPULimit = val
 	} else {
-		vaultConfig.AgentCPU = resource.MustParse("100m")
+		vaultConfig.AgentCPULimit = resource.MustParse("100m")
 	}
 
-	if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-memory"]); err == nil {
-		vaultConfig.AgentMemory = val
+    if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-cpu"]); err == nil {
```

Comment:
Could you just leave a comment on this one that it is preserved for backward compatibility? The same for the memory label. The same should be reflected in the docs PR.